examples/passporting/README.md

```diff
@@ -1 +1,199 @@
 # idOS Passporting Demo
+
```

Comment:
I feel like this whole file is overly verbose. If I wanted this level of detail, I think I'd rather read the code.

But I'm also not super sure what we should want in this file. So, I'm keeping it for now. :+1: